src/renderer/views/Settings/Settings.css

```
@@ -21,6 +21,19 @@
 }
 
+.settingsWindow.maximized {
+  max-inline-size: 100vw;
+  max-block-size: 100vh;
+  border-width: 0;
+  border-radius: 0;
+}
+
+/* The app page keeps its own scrollbar behind the modal. Do not let it show
+   through beside the settings content while the modal covers the viewport. */
+:global(body:has(.settingsWindow.maximized) > .os-scrollbar) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)